.github/workflows/deploy-xtgeo-pypi.yml

```diff
@@ -33,6 +55,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
+        python-version: [3.8]
```

Comment:
May I ask why it is necessary to determine this? The deploy should run for all `CIBW_BUILD`, correct? I get why it would be necessary to determine os to run on, and I would guess that cibuild handles which of the values in `CIBW_BUILD` to use, but then I would also assume that either one would define all python versions possible in `CIBW_BUILD`, or one would not need to define any.

Reply:
Mac OS default Python is 2.7 which does not work with cibuildwheel (and "outer" python is used to install all the cibuildwheel python images)

Reply:
Alright, wanna add a comment on that? simply specify that `cibuildwheel` requires a python 3 version should be sufficient

Reply:
There is a comment line now